src/main/java/com/google/devtools/build/lib/rules/objc/MultiArchSplitTransitionProvider.java

```diff
@@ -144,7 +144,7 @@ public SplitTransition create(AttributeTransitionData data) {
       } else {
         minimumOsVersion = Optional.of(DottedVersion.fromString(minimumOsVersionString));
       }
-    } catch (IllegalArgumentException exception) {
+    } catch (IllegalArgumentException | DottedVersion.InvalidDottedVersionError exception) {
```

Comment:
When is IllegalArgumentException being thrown now?

Reply:
`#getPlatformType` can do it.

Reply:
Would you mind cleaning that up as well? Looks like it can actually be private, so it's just a small change in this file.